backend/lib/config/testing.yaml

```diff
@@ -1,6 +1,7 @@
 port: 3030
 logLevel: info
 logFormat: text
+apiServerUrl: https://minikube:8443
```

Comment:
is this still used? as we have the same value in *backend/lib/config/gardener.js*